internal/export/distro/registrations_test.go

```diff
@@ -203,6 +231,9 @@ func BenchmarkProcessEvent(b *testing.B) {
 	event := models.Event{}
 	event.Device = "dummyDev"
 
+	msg := msgTypes.MessageEnvelope{}
+	msg.Payload, _ = json.Marshal(event)
+
```

Comment:
Need to set content type?

Reply:
+1 
There remains an opportunity to add a separate test that intentionally does not set ContentType. Trevor implemented [processMessage ](https://github.com/edgexfoundry/edgex-go/pull/1356/files/c5f9cc9d00e837fbb0269e0bf973f0439e3bd61f#diff-0d5a195fe75fb296358cc16865eb31d3R175) that responds to this condition accordingly.

Reply:
Added content-type to test.